app/models/claim/base_claim.rb

```diff
@@ -124,7 +124,22 @@ class BaseClaim < ActiveRecord::Base
 
     accepts_nested_attributes_for :basic_fees,        reject_if: :all_blank, allow_destroy: true
     accepts_nested_attributes_for :misc_fees,         reject_if: :all_blank, allow_destroy: true
-    accepts_nested_attributes_for :expenses,          reject_if: :all_blank, allow_destroy: true
+    accepts_nested_attributes_for :expenses,
+      # vat_amount comes in as "0.00" from form so explicitly omit it when
+      # checking blankness.
+      reject_if: proc { |atts|
+        atts["date_dd"].blank? &&
+          atts["date_mm"].blank? &&
+          atts["date_yyyy"].blank? &&
+          atts["expense_type_id"].blank? &&
+          atts["location"].blank? &&
+          atts["distance"].blank? &&
+          atts["reason_id"].blank? &&
+          atts["reason_text"].blank? &&
+          atts["hours"].blank? &&
+          atts["amount"].blank?
+      },
+      allow_destroy: true
     accepts_nested_attributes_for :disbursements,     reject_if: :all_blank, allow_destroy: true
```

Comment:
I think we need tests for this - I suspect that if we fill in a value in the vat amount and none of the other fields, the record will not be created.